### PR TITLE
Added <legend> to <fieldset> element 

### PR DESCRIPTION
**Pull request by NidhiKJha**
This pull request is to add the `<legend>` of a `<fieldset>` element which provides a label

### CODE REVIEW

File: app/javascript/components/form/fields/checkbox-input.jsx [hunk 1]
```diff
@@ -23,7 +23,9 @@ const CheckboxInput = ({ commonInputProps, options, metaInputProps }) => {
       error={error}
       className={css.checkboxContainer}
     >
-      <FormLabel required={required}>{label}</FormLabel>
+      <FormLabel component="legend" required={required}>
```
Comment (jtoliver) [inline app/javascript/components/form/fields/checkbox-input.jsx:26]: Does this change how the label looks? If not, this is ok

Comment (NidhiKJha) [reply app/javascript/components/form/fields/checkbox-input.jsx:26]: I don't think so. But  I will verify it. Thanks

Comment (NidhiKJha) [reply app/javascript/components/form/fields/checkbox-input.jsx:26]: No, it doesn't change the look

Comment (jtoliver) [reply app/javascript/components/form/fields/checkbox-input.jsx:26]: ok cool, I'll merge first thing tomorrow

**Pull request merged**